modules/data.atmosphere/R/solar_angle.R

```diff
@@ -2,6 +2,50 @@
 #'
 #' For explanations of formulae, see https://web.archive.org/web/20180307133425/http://www.itacanet.org/the-sun-as-a-source-of-energy/part-3-calculating-solar-angles/
 #'
+#' @description
+#' `cos_solar_zenith_angle` returns the cosine of solar zenith angle.
+#' The cosine of the solar zenith angle (cossza) is the angle between the sun’s rays and the vertical.
+#' This angle is important because it determines the intensity of solar radiation that reaches the Earth's surface.
+#' When the solar zenith angle is small (i.e. the sun is close to directly overhead), the intensity of solar radiation is high.
+#' When the solar zenith angle is larger (i.e. the sun is closer to the horizon), the intensity of solar radiation is lower, resulting in cooler temperatures.
+#' It is required for photolysis.
+#'
+#' It depends upon the following -
+#'
+#' doy Day of year : It is the day when you are calculating the angle.
+#' January 1st being day 1 and December 31st being day 365 (or 366 in leap years).
+#' It is used to determine the position of the sun in the sky throughout the year.
+#' Unit - Dimensionless (a simple count of the day within a year).
```

Comment:
Use roxygen2 tags for this
```suggestion
#'
#' @param doy Day of year : It is the day when you are calculating the angle.
#' January 1st being day 1 and December 31st being day 365 (or 366 in leap years).
#' It is used to determine the position of the sun in the sky throughout the year.
#' Unit - Dimensionless (a simple count of the day within a year).
```

Reply:
Further clarification: you already have a `@param` tag for every variable, and you don't want to create duplicates. Instead, you want to add the basic variable-by-variable details to the existing variable descriptions (which are too short). Beyond that, if you need to provide a more detailed explanation of what the function is doing, it should go in a `@details` section, not in the `@description`, which should be no more than a few sentences.